.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "npm"
```

Comment:
Given that most dependency upgrade PRs will be spam (e.g. causing breaking changes), I would like to disable the `npm` integration.

Reply:
@mrueg ping 🙂

Reply:
Sorry for the delay, I remove this one.